scripts/SREP/update-upgrade-policy-state/metadata.yaml

```diff
@@ -0,0 +1,30 @@
+file: script.sh
+name: update-upgrade-policy-state
+description: Update the state of the current cluster's upgrade policy using the desired state.
+author: sbai
+allowedGroups: 
+  - SREP
+  - CSSRE
```

Comment:
```suggestion
  - CSSRE
```
I think CSSRE is LPSRE now?

Reply:
Looking at older PRs like these https://github.com/openshift/managed-scripts/pull/105/files , I guess you are right. LPSRE and CSSRE are being considered different.

Reply:
Yeah, we can purge all the CSSRE references, only LPSRE stands true

Reply:
So, I assume it's all good for the current setting of the allowed groups. Please let me know if anything I misunderstood.

Reply:
We can remove the CSSRE reference and can just stick to LPSRE.